students/synowiec_krzysztof/lesson_06_dicts_tuples_sets_args_kwargs/list_to_fictionary_function_for_fantasy_game_inventory.py

```diff
@@ -0,0 +1,21 @@
+from fantasy_game_inventory import display_inventory
+
+
+def main():
+    inv = {'gold coin': 42, 'rope': 1}
+    dragon_loot = ['gold coin', 'dagger', 'gold coin', 'gold coin', 'ruby']
+    inv = add_to_inventory(inv, dragon_loot)
+    display_inventory(inv)
+
+
+def add_to_inventory(inventory, added_items):
+    for item in added_items:
+        if item not in inventory:
```

Comment:
It's fine, but maybe it would be a good idea to use `.get()` here? Something like `inventory.get(item, 0) + 1`
very useful when dealing with python dict
Some https://stackoverflow.com/questions/11041405/why-dict-getkey-instead-of-dictkey